demos/sev-demo/README.md

```diff
@@ -0,0 +1,192 @@
+# CCv0 with AMD SEV
+
+CCv0 can be used with AMD SEV. Check out this [demo video](https://youtu.be/SXZ-hzhjahU) to see it in action. This guide will explain how to take an SEV node and use CCv0 with it. This guide does not cover configuring an SEV machine. You will need to have a node where you can start SEV VMs. This guide also does not cover setting up CCv0. You will need a working deployment of CCv0. You will update this deployment to use SEV. 
+Soon many of these steps will be automated via [Kata Deploy](https://github.com/kata-containers/kata-containers/tree/main/tools/packaging/kata-deploy) and the [Confidential Containers Operator](https://github.com/confidential-containers/operator).
+
+## OVMF
+
+You will need to use OVMF. Furthemore, you will need to use the AMDSEV build of OVMF. An appropriate firmware binary is available in this directory.
```

Comment:
Missing link ? [this directory]?

Reply:
The file is in the same directory as the README. I can add a link to it, though.